lib/school_house_web/templates/error/translation_missing.html.eex

```diff
@@ -0,0 +1,8 @@
+<div class="bg-white">
+  <div class="max-w-2xl px-4 py-16 mx-auto text-center sm:py-20 sm:px-6 lg:px-8">
+    <h2 class="text-3xl font-extrabold text-brand-purple sm:text-4xl">
+      <span class="block">Translation unavailable</span>
```

Comment:
I'm going to wait on the gettext stuff for right now. Then open up some issues and work through it in a separate PR.